Description: Explicitly set old behaviour for cmake policy CMP0026. Required for cmake 3.
Author: Andrew Ross <andrewross@users.sourceforge.net>
Last-Update: 2014-09-17
Origin: upstream

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -54,6 +54,10 @@
   message(STATUS "Explicitly setting policy CMP0023 to OLD")
   cmake_policy(SET CMP0023 OLD)
 endif(POLICY CMP0023)
+if(POLICY CMP0026)
+  message(STATUS "Explicitly setting policy CMP0026 to OLD")
+  cmake_policy(SET CMP0026 OLD)
+endif(POLICY CMP0026)
 
 # It is a fatal error if no working C compiler is available to build
 # the PLplot core C library and core C examples.  All other compilers
--- a/examples/CMakeLists.txt
+++ b/examples/CMakeLists.txt
@@ -397,6 +397,11 @@
     message(STATUS "Explicitly setting policy CMP0023 to OLD")
     cmake_policy(SET CMP0023 OLD)
   endif(POLICY CMP0023)
+  # This should be fixed, but silences warnings for now
+  if(POLICY CMP0046)
+    message(STATUS "Explicitly setting policy CMP0046 to OLD")
+    cmake_policy(SET CMP0046 OLD)
+  endif(POLICY CMP0046)
 
   message(STATUS "CMAKE_PLATFORM_INFO_DIR = ${CMAKE_PLATFORM_INFO_DIR}")
   # Locations where PLplot cmake build system first looks for cmake modules.
